crates/ruff_linter/src/test.rs

```diff
@@ -177,7 +177,7 @@ pub(crate) fn test_contents<'a>(
             transformed = Cow::Owned(transformed.updated(fixed_contents, &source_map));
 
             let program =
-                ruff_python_parser::parse_unchecked_source(source_kind.source_code(), source_type);
+                ruff_python_parser::parse_unchecked_source(transformed.source_code(), source_type);
```

Comment:
Um, this is actually a bug. Sorry, it's difficult to separate all this changes very carefully